xlators/mgmt/glusterd/src/glusterd-brick-ops.c

```diff
@@ -1689,7 +1682,6 @@ glusterd_remove_brick_validate_bricks(gf1_op_commands cmd, int32_t brick_count,
     glusterd_conf_t *priv = THIS->private;
```

Comment:
Hmm... perhaps this should be initialized later, after line 1684?

Reply:
Ack.